main.go

```diff
@@ -83,27 +96,43 @@ func main() {
 	if *debug {
 		log.SetLevel(log.DebugLevel)
 	}
+	// Empty context for all requests in sequel
+	ctx := context.Background()
+	transport := http.DefaultTransport
+	if *insecure {
+		transport = &http.Transport{
+			TLSClientConfig: &tls.Config{
+				InsecureSkipVerify: true,
+			},
+		}
+	}
 
+	wrapTransport := registry.WrapTransport(transport, *registryURL,
+		*username, *password)
 	// Create registry object
-	r, err := client.NewRegistry(*registryURL, nil)
+	r, err := client.NewRegistry(ctx, *registryURL, wrapTransport)
```

Comment:
Context is not used anymore in Docker API. See this: https://github.com/fraunhoferfokus/deckschrubber/commit/9931cc7a4f4cdc515c1dca60c89e875f6f11d889